.maintain/gitlab/check_polkadot_companion_build.sh

```diff
@@ -37,21 +37,32 @@ from this pull request. otherwise, it will uses master instead
 
 EOT
 
+# FIXME: update jq in the CI file
+
+jq="$PWD/jq16"
+curl -sqL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o "$jq"
```

Comment:
need to address this before merging on the CI image

awaiting for https://github.com/paritytech/ci_cd/issues/193